enthosting.py

```diff
@@ -0,0 +1,112 @@
+import asyncio
+from playwright.async_api import async_playwright, expect
+
+# Configuration variables
+BASE_URL = "https://www.entgaming.net/"
+USERNAME = "okIbTimeToPlay"
+PASSWORD = "bCl8v5T8DxbzgG8cFsVUC"
```

Comment:
Don't store password here, maybe os.environ ? but i think it meeeeeeeeh on bot restart dunno, ideas ?

Reply:
I could add it to my startup script, like `ENT_PASSWORD=... python main.py`. Maybe better than having it on public github yeah, I'm down to do that if you are.

Reply:
That's kinda mehhhhhhhhhhhhhh, i could also do it, but what about using environment variable instead ?
Btw, how about switching to docker container for your deployment also ? i've been running mine as this for almost a year and it's working flawlessly : 
![image](https://github.com/user-attachments/assets/4e911ede-be19-4e9b-b3fd-96c505615567)